.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+### Summary
+<What's this change about?>
+
+### Test Plan
+<How did you test your change?>
```

Comment:
It would be nice to give some instructions here on how to run unit tests, and maybe how to run the pre-commit tests?